PomoTodo/PomoTodo/Sources/Views/SettingView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  SettingView.swift
+//  PomoTodo
+//
+//  Created by 심근웅 on 2/18/25.
+//
+
+import SwiftUI
+
+struct SettingView: View {
+  @StateObject var pomoVM: PomoViewModelSetting = PomoViewModelSetting()
+  @State private var isEditMode: Bool = false // 편집모드
+  
+  var body: some View {
+    NavigationView {
+      List {
+        // 뽀모도로 타이머 섹션
+        Section(header: Text("뽀모도로 설정")) {
+          ForEach($pomoVM.timers, id: \.index) { pomo in
+            NavigationLink(destination: PomoDetailSettingView(pomo: pomo).environmentObject(pomoVM)) {
+              PomoSettingRow(pomo: pomo).environmentObject(pomoVM)
+            }
+          }
+        }
+//        .modifier(HeaderMdifier(fontsize: 15))
+        // 투두 태그 섹션
+        Section(header:TagSettingHeader(isEditMode: $isEditMode).environmentObject(pomoVM)) {
+          ForEach($pomoVM.options, id: \.id) { tag in
+            TagSettingRow(option: tag, isEditMode: $isEditMode)
```

Comment:
뷰마다 .environmentObject(pomoVM)를 할 필요는 없습니다.
다 제거 하시고 NavigationView에 하시면될 것 같습니다.
```suggestion
NavigationView {
  // code ..
}.environmentObject(pomoVM)
```